seleniumJSEnv/src/test/scala/org/scalajs/jsenv/selenium/KeepAliveTest.scala

```diff
@@ -18,7 +18,7 @@ class KeepAliveTest {
   private final class MockWebDriver extends WebDriver with JavascriptExecutor {
     var closed = false
 
-    def close(): Unit = closed = true
+    def close(): Unit = ???
```

Comment:
Should this be moved below under the "Stuff we won't need" comment?

Reply:
Yes :)